build/.goreleaser.yml

```diff
@@ -40,7 +43,91 @@ builds:
       - -s -w -X main.gitCommit={{.Commit}}
     goarch:
       - amd64
+###
+###  arm for fpm
+###
+  - id: linux-newrelic-infra-fpm-arm
```

Comment:
is there a reason for why do we need this section? couldn't we just add `arm` in goarch from previous *-fpm build?

Reply:
different nfpms :/ could we mange it with some conditions, to use only amd for amd and arm for arm? 🤔

Reply:
goreleaser should handle this right? just mention the same build id in nfpm

Reply:
ok, I see, If we don't do it like this, would create extra packages